apps/mull-ui/src/app/components/navigation-bar/navigation-bar.scss

```
@@ -2,17 +2,20 @@
 
 .nav-container {
   background: $background-color;
```

Considering the changes you've done to how the navbar is styled (fixed position over the content and the content needs padding to avoid overlap), id recommends you set a fixed height for the navbar and then set the same value for the top padding (to make things easier you can make it a variable). 

```suggestion
  box-sizing: border-box;
  height: $nav-bar-height-mobile;
  background: $background-color;
  @media (min-width: $breakpoint-mobile) {
      height: $nav-bar-height-desktop;
  }
```